vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Install and use packages on Linux with vcpkg
+description: Tutorial guides the user through the process of installing and using packages on Linux with vcpkg.
+author: JavierMatosD
+ms.author: javiermat
+ms.topic: tutorial
+ms.date: 08/23/2023
+ms.prod: vcpkg
+#CustomerIntent: As a beginner C++ developer, I want to learn how to install and manage packages on a Linux environment using vcpkg, so that I can easily set up and maintain C++ projects with necessary dependencies.
+---
+
+# Tutorial: Install and use packages on Linux with vcpkg
+
+This tutorial shows you how to set up a C++ "Hello World" CMake project using the `fmt` library. Manage dependencies, construct the application, and execute the project with the help of vcpkg. This guide is tailored for a Linux environment.
+
+## Prerequisites
+
+- A terminal to work in.
```

Comment:
'A terminal to work in' is very vague. Do you want to say 'A terminal such as Zsh or WSL'